src/sentry/api/endpoints/project_details.py

```diff
@@ -109,22 +90,63 @@ def validate(self, data):
 
 
 class ProjectMemberSerializer(serializers.Serializer):
-    isBookmarked = serializers.BooleanField()
-    isSubscribed = serializers.BooleanField()
+    isBookmarked = serializers.BooleanField(
+        help_text="Enables starring the project within the projects tab. Can get updated with `project:read` permission.",
```

Comment:
nit: bold scopes to look like rest of scopes on page
```suggestion
        help_text="Enables starring the project within the projects tab. Can be updated with **`project:read`** permission.",
```